src/drakvuf.cpp

```diff
@@ -286,6 +286,19 @@ void drakvuf_c::resume()
     drakvuf_resume(drakvuf);
 }
 
+void drakvuf_c::toggle_context_interception(GSList* processes)
+{
+    GSList* process = processes;
+
+    while (process != NULL)
+    {
+        drakvuf_intercept_process_add(this->drakvuf, (char*)process->data, 0, false);
```

Comment:
It would be helpful if the user could specify either process name or pid from the command line. Right now only process name is possible. I don't think we need separate command-line options for this, if the input is just a number I think it would be safe to assume its a PID.

Reply:
Indeed it would be helpful, but rather than allowing the user to specify PID only instead of the name, It would be safer to allow the user to specify the PID alongside the name because it is more likely to have another process with the same PID in a short amount of time but less likely that it will have the same name and PID; therefore the argument for this option could be separated using a delimiter `process_name, PID`, and the user will have the option to specify either the name only or name & PID.

Reply:
Sure, but I particularly want to be able to use just PID. Instead of comma-delimiter I would suggest something like `process-name:pid`. If no `:` is present treat it as process-name non-strict mode. If `:` is present but nothing before it, then use just pid, like `:1234`.